src/main/java/ch/njol/skript/expressions/ExprItemWithEnchantmentGlint.java

```diff
@@ -0,0 +1,64 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Item with Enchantment Glint")
+@Description("Get an item with or without enchantment glint.")
+@Examples({
+	"set {_item with glint} to diamond with enchantment glint",
+	"set {_item without glint} to diamond without enchantment glint"
+})
+@RequiredPlugins("Spigot 1.20.5+")
+@Since("INSERT VERSION")
+public class ExprItemWithEnchantmentGlint extends PropertyExpression<ItemType, ItemType> {
+
+	static {
+		if (Skript.isRunningMinecraft(1, 20, 5))
+			Skript.registerExpression(ExprItemWithEnchantmentGlint.class, ItemType.class, ExpressionType.PROPERTY, "%itemtypes% with[:out] [enchant[ment]] glint");
+	}
+
+	private boolean glint;
+
+	@Override
+	@SuppressWarnings("unchecked")
+	public boolean init(Expression<?>[] expressions, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		setExpr((Expression<ItemType>) expressions[0]);
+		glint = !parseResult.hasTag("out");
+		return true;
+	}
+
+	@Override
+	protected ItemType[] get(Event event, ItemType[] source) {
+		return get(source.clone(), itemType -> {
```

Comment:
Is the intention here to clone the item types? I don't believe this will actually clone them.

Reply:
I saw one cloned the ItemTypes while I was looking on the usages of PropertyExpression to create this.

Reply:
Cloning is correct, but I believe you'll need to call clone on each ItemType rather than the array